src/connectors/github_organization.py

```diff
@@ -0,0 +1,339 @@
+from json import dumps
```

Comment:
At the top of each connector is a docstring that gives  the connector a human-readable name and a description that appear in the WebUI and succinctly explain what kind of data the user is to expect.